lib/writer.js

```diff
@@ -720,7 +720,7 @@ async function encodeRowGroup(schema, data, opts) {
 function encodeFooter(schema, rowCount, rowGroups, userMetadata) {
   let metadata = new parquet_thrift.FileMetaData()
   metadata.version = PARQUET_VERSION;
-  metadata.created_by = 'parquet.js';
+  metadata.created_by = '@dsnp/parquetjs';
```

Comment:
I feel we are drifting far enough that we should tag the files specifically for our package